ruby/vampires.rb

```diff
@@ -1,3 +1,4 @@
+
 #ask employee questions to determine if they're a vampire
 
 	def vampire_test
```

Comment:
I love seeing students practice encapsulating actions via methods before they are introduced in the curriculum!  Nice work.